tests/testthat/test-compare.R

```diff
@@ -289,6 +289,29 @@ test_that("can compare R6 objects", {
   })
 })
 
+test_that("can compare S7 objects", {
+  A <- S7::new_class("A", properties = list(a = S7::class_numeric))
```

Comment:
We may want a `skip_if_not_installed("S7")` here.

Reply:
We don't generally bother with that unless the package is hard to install for some reason.